jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -1140,6 +1155,14 @@ properties:
             type: integer
             fits_keyword: NRIMDTPT
             blend_table: True
+          nod_type:
```

Comment:
I assume you have the source for these lists, but I would've guessed there'd be a 2- and 5-POINT-NOD in addition to the 3-POINT-NOD listed here. Any idea what gives?

Reply:
This keyword only applies to NIRSpec MOS exposures where the names of the nod pattern use the x-SHUTTER-SLITLET syntax (nods within a slitlet). The PATTTYPE keyword has values like 2-POINT-NOD, 3-POINT-NOD, etc., which is used for NIRSpec FS and other modes that are nodding within a physical slit (as opposed to a slitlet construct). Yes, the values shown here come from what's currently in the JWST Keyword Dictionary, which is based on all known values defined in PPS.